### ISSUE TITLE
Thumb not resizing to requested geometry

### ISSUE THREAD

**Issue by mrmanishs** [ruby-av/paperclip-av-transcoder#9]
Hi there,

I have the following in my paperclip definition:
  has_attached_file :video, 
  :default_url => '/filetype-icons/filetype-unknown.png', 
  :styles => {
      :thumb => { :geometry => "32x32>", :format => 'png', :time => 10 }
  }, 
  :path => ":rails_root/public/apartment_files/:attachment/:id_partition/:style/:filename", 
  :url => "/apartment_files/:attachment/:id_partition/:style/:filename",
  :processors => [:transcoder]
 

However, when transocding, the thumb does not transcode to 32x32. Instead it goes to 1920x1080. What could I be doing wrong?

**Comment by cjeffries** [comment]
Currently the gem doesn't include the geometry options in the conversion output. To include it in there you can change your code to:

```
has_attached_file :video, 
:default_url => '/filetype-icons/filetype-unknown.png', 
:styles => {
:thumb => { :geometry => "32x32>", :format => 'png', :time => 10, :convert_options => {:output => {:s => "32x32"}} }
}, 
:path => ":rails_root/public/apartment_files/:attachment/:id_partition/:style/:filename", 
:url => "/apartment_files/:attachment/:id_partition/:style/:filename",
:processors => [:transcoder]
```

I'm yet to test if this will also adhere to the !#<> options you provide in the geometry option.

**Comment by owahab** [comment]
Fixed in #11

**Comment by mrmanishs** [comment]
This works perfectly. Thank you for the fix.
